src/components/molecules/LogOutput/FullscreenLogOutput/FullScreenLogOutput.tsx

```diff
@@ -1,6 +1,7 @@
 import {MouseEvent, forwardRef, useCallback, useEffect, useRef, useState} from 'react';
 
 import Ansi from 'ansi-to-react';
+import debounce from 'lodash.debounce';
```

Comment:
Optionally, to avoid multiple packages:
```suggestion
import {debounce} from 'lodash';
```